src/columntable.jl

```diff
@@ -95,3 +95,39 @@ function withinitem(nitem, df; tag = 'I')
     value.item = repeat(PooledArray(nlevels(nitem, tag=tag)), outer=nrow)
     value
 end
+
+"""
+    power_table(sim, 0.05)
+
+Returns a `DataFrame` with two columns, `coefname` and `power`, with the proportion of 
+simulated p-values less than alpha, for `sim`, the output of `simulate_waldtests`.
+"""
+
+function power_table(sim, alpha = 0.05)
+    pvals = DataFrame(columntable(sim).p)
+    pvals = stack(pvals, 1:ncol(pvals)) 
```

Comment:
I think the `1:ncols(pvals)` is redundant, i.e. that `stack` does all the columns by default.

Reply:
Yup, I'll fix that.